.github/workflows/coverage.yml

```diff
@@ -0,0 +1,80 @@
+name: coverage-linux
+on: [push, pull_request]
+jobs:
+  build:
+    name: ${{ matrix.name }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: flambda2_coverage
+            config: --enable-middle-end=flambda2 --enable-coverage
+            ocamlparam: ''
+
+    env:
+      J: "3"
+
+    steps:
+    - name: Checkout the Flambda backend repo
+      uses: actions/checkout@master
+      with:
+        path: 'flambda_backend'
+
+    # This adds a switch at $GITHUB_WORKSPACE/_opam
+    - name: Set up OCaml 4.12
+      uses: ocaml/setup-ocaml@v2
+      with:
+        ocaml-compiler: 4.12
+        opam-pin: false
+        opam-depext: false
+
+    - name: Get host build environment from cache
+      uses: actions/cache@v1
+      id: cache
+      with:
+        path: ${{ github.workspace }}/_opam
+        key: ${{ matrix.os }}-cache-ocaml-412-dune-2.9.1-g7606d586
+
+    - name: Build Dune
+      if: steps.cache.outputs.cache-hit != 'true'
+      run: |
+        opam pin dune --yes \
+          git+https://github.com/ocaml-flambda/dune#special_dune
+
+    - name: Build bisect_ppx
+      if: steps.cache.outputs.cache-hit != 'true'
+      run: |
+        # bisect_ppx fails to build with current ppxlib
+        opam pin ppxlib 0.23.0 --yes
+        opam pin bisect_ppx --yes \
+          git+https://github.com/lukemaurer/bisect_ppx#html-report-collate-fix
+
+    - name: Trim cached files
+      if: steps.cache.outputs.cache-hit != 'true'
+      run: |
+        # Remove unneeded parts to shrink cache file
+        rm -rf _opam/{lib/ocaml/expunge,bin/*.byte}
+
+    - name: Configure Flambda backend
+      working-directory: flambda_backend
+      run: |
+        autoconf
+        ./configure \
+          --prefix=$GITHUB_WORKSPACE/_install \
+          --with-dune=$GITHUB_WORKSPACE/_opam/bin/dune \
+          ${{ matrix.config }}
+
+
+    - name: Build, install and test Flambda backend
+      working-directory: flambda_backend
+      run: opam exec -- make ci
+      env:
+        BUILD_OCAMLPARAM: ${{ matrix.ocamlparam }}
+
+    - name: Publish coverage report
```

Comment:
Where exactly does the report end up?

Reply:
The answer turns out to be in an "artifacts" place associated with Github Actions.